packages/phone-number-privacy/combiner/src/signing/get-threshold-signature.ts

```diff
@@ -158,11 +158,19 @@ async function requestSignatures(request: Request, response: Response) {
 
   logResponseDiscrepancies(responses, logger)
   const majorityErrorCode = getMajorityErrorCode(errorCodes, logger)
-  if (!blsCryptoClient.hasSufficientVerifiedSignatures()) {
+  if (!blsCryptoClient.hasSufficientSignatures()) {
```

Comment:
I think it's okay to leave this here, primarily because it's self-documenting. It helps the reader understand when we'd expect the object to return a value and when to not bother trying.